### DISCUSSION TITLE
Merge without renaming

### DISCUSSION THREAD

**Discussion by SarahLQuinn** [satijalab/seurat#7771] [Q&A] [answered]
I am trying to combine two count matrixes from the same sample (sequenced it twice for greater depth). The Seurat merge function does almost what (combine raw count matrices) I want but automatically renames identical barcodes to keep them separate. Is there a way to have it not rename and instead sum the overlapping barcode matrixes. Or another clever solution to combine re-sequenced samples post-alignment to avoid going back and concatenating the fastQs? Thanks in advance for any help!

**Comment by samuel-marsh** [comment, marked as answer]
Hi,

So combining two sequencing runs of the same library should not be combined at matrix level.  The reason is because you can’t account for duplicated counts via UMIs.  This needs to be done at level of aligner (I.e. CellRanger).  You don’t need to concatenate the fastqs but simply supply both sets of fastq for the sample during alignment.  This way it will properly deal with UMIs.

Best,
Sam
